### PR TITLE
medspacy_quickumls 3.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/Georgetown-IR-Lab/QuickUMLS/blob/master/LICENSE.txt
Requirements:
- https://github.com/Georgetown-IR-Lab/QuickUMLS/blob/master/setup.py
- https://github.com/Georgetown-IR-Lab/QuickUMLS/blob/master/requirements.txt

Notes:
- Release 3.0 exists on PyPi https://pypi.org/project/medspacy-quickumls/3.0/. It has requirements.txt with:
```
    numpy>=1.8.2
    spacy>=3.1.3
    unidecode>=0.4.19
    nltk>=3.3
    pysimstring
    unqlite>=0.8.1
    pytest>=6
    six
```

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
```
+ pip check
thinc 8.0.15 has requirement pydantic!=1.8,!=1.8.1,<1.9.0,>=1.7.4, but you have pydantic 1.10.2.
spacy 3.3.1 has requirement pydantic!=1.8,!=1.8.1,<1.9.0,>=1.7.4, but you have pydantic 1.10.2.
```

**Review by akabanovs — APPROVED** [review]
LGTM. Need to remove abs.yaml file?

File: recipe/meta.yaml
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: would it be usable for customer then ?

Comment (skupr-anaconda) [reply recipe/meta.yaml]: You are right. I'll try to rebuild `thinc` and `spacy` and also update to the latest.

Comment (skupr-anaconda) [reply recipe/meta.yaml]: To fix the issue I'm updating spacy to 3.5.3 https://github.com/AnacondaRecipes/spacy-feedstock/pull/19

Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Review by ViridianMelody — APPROVED** [review]
LGTM as long as the abs.yaml is removed.

**Review by varlackc — COMMENTED** [review]
We might want to remove the `abs.yaml` file before merging to defaults

**Pull request merged**
